src/Symfony/Contracts/composer.json

```diff
@@ -23,9 +23,17 @@
         "psr/container": "^1.0",
         "symfony/polyfill-intl-idn": "^1.10"
     },
+    "replace": {
+        "symfony/cache-contracts": "self.version",
```

Comment:
Is the idea to set up a subsplit to extract individual contract packages? If not, that would not work since all the contracts still would be installed into vendor, indexed by IDE and interfere with code suggestions.

Reply:
Yes, by splitting each subdirectory in a package.
This one is for `symfony/contracts`, which must replace all individual packages when it is installed.
This is what we do for symfony/symfony and symfony/security for example.
I don't understand your objection.

Reply:
That's fine. I just thought there will be no individual packages that do not install just what is needed without extras.